aries-core/src/js/components/core/Selector/SelectorHeader.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Text, ThemeContext } from 'grommet';
+import { Checkmark } from 'grommet-icons';
+import { SelectorGroupContext } from './SelectorGroup';
+
+const SelectorIndicator = ({ selected, indicator, ...rest }) => {
+  const { multiple } = useContext(SelectorGroupContext);
+  const theme = useContext(ThemeContext);
+
+  const { round = !multiple ? 'full' : 'xxsmall', size = 'medium' } = indicator;
+
+  return (
+    <Box
+      border={{
+        color: selected ? 'brand' : 'border',
+        size: 'xsmall',
+      }}
+      round={round}
+      height={theme.global.edgeSize[size]}
+      width={theme.global.edgeSize[size]}
+      justify="center"
+      align="center"
+      background={selected ? 'brand' : undefined}
+      flex={false}
+      {...rest}
+    >
+      {selected && (
+        // need to talk with Lucas
```

Comment:
I'm not opposed to this, I like how it looks visually, but it seems like we lose a little of the styling similarities between radio button and single selection. This change might not make it is as clear for the user that the circle is a single select.

I agree it would be good to check with Lucas

Reply:
yes Im planning tomorrow to discuss